agent/xds/delta.go

```diff
@@ -95,33 +95,26 @@ func (s *Server) DeltaAggregatedResources(stream ADSDeltaStream) error {
 func getEnvoyConfiguration(proxySnapshot proxysnapshot.ProxySnapshot, logger hclog.Logger, cfgFetcher configfetcher.ConfigFetcher) (map[string][]proto.Message, error) {
 	switch proxySnapshot.(type) {
 	case *proxycfg.ConfigSnapshot:
-		logger.Trace("ProxySnapshot update channel received a ProxySnapshot of type ConfigSnapshot",
-			"proxySnapshot", proxySnapshot,
-		)
+		logger.Trace("ProxySnapshot update channel received a ProxySnapshot of type ConfigSnapshot")
```

Comment:
should we add a comment to not log the proxySnapshot object in case someone thinks that might be a good idea in the future?